### PR TITLE
Added webauthn based registration feature

### PR DESCRIPTION
**Pull request by karthikaselvaraj-okta**
Have added webauthn based registration feature for automation testing

### CODE REVIEW

File: samples/embedded-auth-with-sdk/src/test/java/info/seleniumcucumber/userStepDefinitions/SelfServiceRegistration.java [hunk 1]
```diff
@@ -305,6 +308,35 @@ public void mary_opens_activate_page_with_token() {
...
+    }
+
+    @And("^she selects WebAuthn from the list$")
+    public void she_selects_webauth_from_the_list() {
+        verifyPage.webAuthnRadioButton.click();
+    }
+
+    @And("^she inputs the fingerprint$")
+    public void she_inputs_the_fingerprint() {
+        System.out.println("WebAuth-Start");
```
Comment (vijetmahabaleshwar-okta) [inline samples/embedded-auth-with-sdk/src/test/java/info/seleniumcucumber/userStepDefinitions/SelfServiceRegistration.java:324]: Remove the prints. We don't need them now that tests are passing

File: samples/embedded-auth-with-sdk/src/test/java/info/seleniumcucumber/userStepDefinitions/SelfServiceRegistration.java [hunk 2]
```diff
@@ -305,6 +308,35 @@ public void mary_opens_activate_page_with_token() {
         driver.get(activateURL);
     }
 
+    //Added for webauthn
```
Comment (vijetmahabaleshwar-okta) [inline samples/embedded-auth-with-sdk/src/test/java/info/seleniumcucumber/userStepDefinitions/SelfServiceRegistration.java:311]: Remove comment. Not useful.

**Review by vijetmahabaleshwar-okta — APPROVED** [review]
Approved with 2 minor comments.

File: samples/embedded-auth-with-sdk/src/test/java/info/seleniumcucumber/userStepDefinitions/SelfServiceRegistration.java (hunk 1 above)
Comment (karthikaselvaraj-okta) [reply samples/embedded-auth-with-sdk/src/test/java/info/seleniumcucumber/userStepDefinitions/SelfServiceRegistration.java]: Thanks for the review. Have removed comments and print statements.

**Pull request closed without merging**